doc/python/axes.md

```diff
@@ -380,6 +380,30 @@ fig.update_layout(title_text="Apple Stock Price")
 fig.show()
 ```
 
+#### Adding minor ticks
+
+_new in 5.8_
+
+You can position and style minor ticks on a Cartesian axis using `minor`. This takes a `dict` of properties to apply to minor ticks. Available properties include: `tickmode`, `tickvals`, `tickcolor`, `ticklen`, `tickwidth`, `dtick`, `tick0`, `nticks`, `ticks`, `showgrid`, `gridcolor`, `griddash`, and `gridwidth`.
+
+In the following example, we add minor ticks to the x-axis and then to the y-axis. For the y-axis we add ticks on the outside: `minor = {"ticks":"outside"}`. On the x-axis we've spcified some additional properties to style the minor ticks, setting the length of the ticks with `ticklen` and the color with `tickcolor`. We've also turned on grid lines for the x-axis minor ticks using `showgrid`.
+
+Note: Minor ticks and grid lines are not currently supported on color bars, ternary plots, polar charts, geo plots, or on multi-categorical, log, or 3D axes.
```

Comment:
`log` is supported. So you could remove it from the list here.

Reply:
But `category` axes also don't support minor.

Reply:
> But `category` axes also don't support minor.

`category` is supported e.g. on [ticklabelposition-5 mock](https://github.com/plotly/plotly.js/blob/master/test/image/mocks/ticklabelposition-5.json).

Reply:
That mock doesn't have minors on category axes... ? https://github.com/plotly/plotly.js/blob/master/test/image/baselines/ticklabelposition-5.png

Reply:
I'd be more interested in how this works with `multicategory` ;)